IoProtocol.h

```diff
@@ -134,18 +134,11 @@ struct      // IOP_TYP1_CMD_DATA
 SDioRequest;
 
 // Request for Digital IO modules: output with variable number of pwm values
-typedef
-#include <COMP_packBegin.h>
-struct      // IOP_TYP1_CMD_DATA2
-{
-    UIoProtocolHeader uHeader;
-    INT16U i16uOutput;
-    INT16U i16uChannels;    // bitfield pwm channel
-    INT8U  ai8uValue[16];   // [0-100] pwm value in %
-    INT8U  i8uCrc;
-}
-#include <COMP_packEnd.h>
-SDioPWMOutput;
+struct pwm_data {
+	u16 output;
+	u16 channels;		// bitfield pwm channel
+	u8 value[16];		// [0-100] pwm value in %
+};
```

Comment:
Hm, don't you need to retain the `__attribute__((__packed__))`?

Reply:
Actually no, there is no padding in this structure. We could add the "packet" attribute for documentation purposes, though...

Reply:
I just added the packed attribute. It should not have any effect on this struct but at least it documents that this structure is indeed packed.

Reply:
Thanks. I seem to recall that the C compiler is free to insert as much padding as it likes anywhere in the struct. Better safe than sorry.